### PR TITLE
Responsive_design 

### PR DESCRIPTION
**Pull request by Dipeshtwis**
## the next web page clone
The project includes :

- media queries (for responsiveness)
- fluid property
- float for the navbar
- grid and flexbox

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,715 @@
+@import "./reset.css";
```
Comment (lucasmazo32) [inline assets/css/style.css:1]: - [x] Hello team, this project requires you to have a Mobile-first approach. Therefore you should not have media queries for small screens. I will paste the screenshot here.

> image

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

It was hard to not approve the project because everything is great, but you did not use a mobile-first approach. 

For the project to get approved, kindly implement the changes mentioned below.

### Happy coding! :computer:

## Lucas Mazo

**Comment by Dipeshtwis** [comment]
The project is updated which is

- based upon mobile-first approach

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED :champagne:
Excellent job on this project guys. :fireworks:  is. Keep it up.
##
***All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.***
- Congratulations on the splendid job @Dipeshtwis & @rida-elbahtouri  :+1:
- You can now merge to master. I wish you all the very best in your next tasks 🥇 
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Pull request merged**